### PR TITLE
Add Pecl YAML parser

### PR DESCRIPTION
**Pull request by mallsbill**

### CODE REVIEW

**Comment by mimmi20** [comment]
@mallsbill Please add some information about this optional dependency in the composer.json.

**Comment by mallsbill** [comment]
> @mallsbill Please add some information about this optional dependency in the composer.json.

@mimmi20 I added ext-yaml to suggest

Last travis job failed on "composer require doctrine/cache ~1.2" on PHP 5.3 build
But it should work if you re-run it

File: Yaml/Pecl.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+namespace DeviceDetector\Yaml;
+
+class Pecl implements Parser
```
Comment (sgiehl) [inline Yaml/Pecl.php]: Would you mind adding a docblock for the class describing the purpose?

File: Yaml/Pecl.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+{
+    public function parseFile($file)
+    {
+        if(function_exists(yaml_parse_file) === false)
+            throw new \Exception('Pecl YAML extension is not installed');
```
Comment (sgiehl) [inline Yaml/Pecl.php]: please add brackets even for one line if statements.

**Review by sgiehl — CHANGES_REQUESTED** [review]
Left some comments, otherwise looks good :+1:

**Review by sgiehl — APPROVED** [review]

**Pull request merged**